xrviz/dashboard.py

```diff
@@ -82,7 +111,12 @@ def create_plot(self, *args):
                 sel_data = sel_data.to_dataset(name=f'{sel_data.name}_')
 
             assign_opts = {dim: self.data[dim] for dim in sel_data.dims}
-            graph = sel_data.assign_coords(**assign_opts).hvplot.quadmesh(**graph_opts)
+            graph = sel_data.assign_coords(**assign_opts).hvplot.quadmesh(**graph_opts).opts(active_tools=['wheel_zoom', 'pan'])
+
+            if self.has_cartopy and is_geo:
+                graph = feature_map * graph
```

Comment:
There may not be any features selected

Reply:
In that case `feature_map` would be empty i.e. `gv.Overlay()`. There would be no error.